src/Uno.UI.Runtime.Skia.MacOS/UnoNativeMac/UnoNativeMac/UNOApplication.h

```diff
@@ -23,6 +23,9 @@ bool uno_application_open_url(const char *url);
 bool uno_application_query_url_support(const char *url);
 bool uno_application_is_bundled(void);
 
+typedef void (*application_start_fn_ptr)(void);
+application_start_fn_ptr uno_get_application_start_callback(void);
```

Comment:
My objC knowledge is very limited, but here you're declaring uno_get_application_start_callback without uno_set_application_start_callback.

Reply:
Yes, on purpose :-)

That, like a lot of the native runtime code, is a C layer over ObjC code.

The `get` function is in the `.m` code as an `inline` function. It's not in the header file since it's not meant to be available externally (to managed code). An _old_ C-way to make things private ;-)